pkg/agent/datamodel/sig_config.go

```diff
@@ -208,6 +208,7 @@ var AvailableWindowsSIGDistros []Distro = []Distro{
 	AKSWindows2019,
 	AKSWindows2019Containerd,
 	AKSWindows2022Containerd,
+	AKSWindows2022ContainerdGen2,
```

Comment:
It is better to separate these code in another PR from the build code.

Reply:
Moved https://github.com/Azure/AgentBaker/pull/2113.